packages/itwin/reports-config-widget/src/widget/components/ExtractionStates/QueuedExtractionState.tsx

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { SvgStatusPending } from "@itwin/itwinui-icons-color-react";
+import { ReportsConfigWidget } from "../../../ReportsConfigWidget";
+
+export const QueuedExtractionState = () => {
+  return (
```

Comment:
Can be made into an implicit return.

Reply:
All `ExtractionState` components now uses implicit return